Modules/Loadable/Markups/VTKWidgets/vtkSlicerAngleRepresentation3D.cxx

```diff
@@ -188,6 +202,8 @@ void vtkSlicerAngleRepresentation3D::UpdateFromMRML(vtkMRMLNode* caller, unsigne
 
   this->UpdateRelativeCoincidentTopologyOffsets(this->LineMapper);
```

Comment:
It would make things safer (harder to forget about occluded mapper) and a bit more compact if `UpdateRelativeCoincidentTopologyOffsets` would take two arguments, the non-occluded and the occluded mapper:

    this->UpdateRelativeCoincidentTopologyOffsets(this->LineMapper, this->LineOccludedMapper);